### ISSUE TITLE
Develop Strategy for Accessibility Testing with People with Lived Experience

### ISSUE THREAD

**Issue by mgifford** [CivicActions/accessibility#55]
We need to test patterns with people with lived experience of disability who can help focus on the barriers that the assistive technology that they use cannot overcome. It is useful to have this feedback to allow the site to exceed bare compliance and produce a site that is actually easier for people with disabilities to accomplish their tasks (or at least no harder than it needs to be). 

Firms that do this include 
- https://makeitfable.com/
- https://knowbility.org/services/usability-testing/
- https://lighthouse-sf.org/design/consulting/

Worth also noting that some of this might be done in-house at some point or through contractors. Having a couple humans available to test projects at various stages of development could be useful. We could it on a consulting basis, have everyone do a training on how we develop projects and how we would have them do accessibility audits. We could get people engaged early by starting to vetting things as soon as we have wireframes and theme frameworks ready for review.

There are pros and cons for going through another company: they would do the vetting; would ideally be training their staff on state of the art testing methods; the flip side is we'd always have an orientation period on any project that might be shortened if we developed our own accessibility testing corps. Security with some government clients may also be a concern than an inhouse team could handle.

**Comment by lukefretwell** [comment]
@mgifford what's the best label for this?

**Comment by mgifford** [comment]
This is a process that needs to be developed.  Added a process tag.  Obviously with a process would be content & likely practice areas.

**Comment by niradatta1** [comment]
Would love to help out with this!  Have we started any documentation on this yet?  Are we at a  point where we have specific patterns we want to user test?

**Comment by mgifford** [comment]
At this point there isn't a document where we are collecting ideas.  

Simplest way would be to just start something draft as a GoogleDoc and link it here.  We can mark it as internal while we are thinking through how we want to approach this.  

We can also publish some ideas and then reach out to the community to see where we can improve it.  The whole point of developing in the open is that we can engage others outside of CivicActions and help form a better practice that what we were able to find before we started.  Or maybe we find something we can't improve on and we simply point to it as a resource we are using and encourage others to as well.

**Comment by mgifford** [comment]
The NFB maintains a 300+ blind user initiative named the BUILD team, ([Blind Users Innovating Leadership and Design](https://nfb.org/programs-services/center-excellence-nonvisual-access/blind-users-innovating-and-leading-design?q=programs-services%2Fcenter-excellence-nonvisual-access%2Fblind-users-innovating-and-leading-design))
Its just an open group of volunteers across skill levels who want to provide feedback centrally to businesses looking for it.
